### ISSUE TITLE
Virtual Machine __refresh overwrites the data incorrectly.

### ISSUE THREAD

**Issue by svt-kmcdaniel** [HewlettPackard/simplivity-python-sdk#36]
The __refresh method incorrectly replaces self.data with the response without stripping off the DATA_FIELD before hand.

Here is the virtual machine data from the get_all method
```
<simplivity.resources.virtual_machines.VirtualMachine object at 0x7f7ba98fd160>
{   'app_aware_vm_status': 'UNKNOWN',
    'cluster_group_ids': <DATA REMOVED>,
    'compute_cluster_parent_hypervisor_object_id': <DATA REMOVED>,
    'compute_cluster_parent_name': <DATA REMOVED>,
    'created_at': '2020-03-17T22:24:08Z',
    'datastore_id': <DATA REMOVED>,
    'datastore_name': <DATA REMOVED>,
    'deleted_at': 'NA',
    'host_id': <DATA REMOVED>,
    'hypervisor_management_system': <DATA REMOVED>,
    'hypervisor_management_system_name': <DATA REMOVED>,
    'hypervisor_object_id': <DATA REMOVED>,
    'hypervisor_type': 'VSPHERE',
    'id': <DATA REMOVED>,
    'name': 'ubuntu-bionic-18.04-cloudimg-20200218',
    'omnistack_cluster_id': <DATA REMOVED>,
    'omnistack_cluster_name': '<DATA REMOVED>,
    'policy_id': <DATA REMOVED>,
    'policy_name': <DATA REMOVED>,
    'replica_set': [   {   'id': <DATA REMOVED>,
                           'role': 'SECONDARY'},
                       {   'id': <DATA REMOVED>,
                           'role': 'PRIMARY'}],
    'state': 'ALIVE'} 
```

Here is the virtual machine data after a refresh occurs
```
<simplivity.resources.virtual_machines.VirtualMachine object at 0x7f7ba98fd160>
{   'virtual_machine': {   'app_aware_vm_status': 'UNKNOWN',
                           'cluster_group_ids': <DATA REMOVED>,
                           'compute_cluster_parent_hypervisor_object_id': <DATA REMOVED>,
                           'compute_cluster_parent_name': <DATA REMOVED>,
                           'created_at': '2020-03-17T22:24:08Z',
                           'datastore_id': <DATA REMOVED>,
                           'datastore_name': <DATA REMOVED>,
                           'deleted_at': 'NA',
                           'ha_resynchronization_progress': 100,
                           'ha_status': 'SAFE',
                           'host_id': <DATA REMOVED>,
                           'host_name': <DATA REMOVED>,
                           'hypervisor_allocated_capacity': 12552624407,
                           'hypervisor_allocated_cpu': -1,
                           'hypervisor_consumed_cpu': 0,
                           'hypervisor_consumed_memory': 0,
                           'hypervisor_cpu_count': 1,
                           'hypervisor_folder_name': <DATA REMOVED>,
                           'hypervisor_free_space': 11460194304,
                           'hypervisor_instance_id': <DATA REMOVED>,
                           'hypervisor_is_template': False,
                           'hypervisor_managed_by': '',
                           'hypervisor_management_system': <DATA REMOVED>,
                           'hypervisor_management_system_name': <DATA REMOVED>,
                           'hypervisor_object_id': <DATA REMOVED>,
                           'hypervisor_total_memory': 1024,
                           'hypervisor_type': 'VSPHERE',
                           'hypervisor_uuid': <DATA REMOVED>,
                           'hypervisor_virtual_disk_count': 1,
                           'hypervisor_virtual_machine_power_state': 'OFF',
                           'id': <DATA REMOVED>,
                           'modified_at': 'Thu Jan 01 00:00:00 GMT 1970',
                           'name': 'ubuntu-bionic-18.04-cloudimg-20200218',
                           'omnistack_cluster_id': <DATA REMOVED>,
                           'omnistack_cluster_name': <DATA REMOVED>,
                           'policy_id': <DATA REMOVED>,
                           'policy_name': <DATA REMOVED>,
                           'replica_set': [   {   'id': <DATA REMOVED>,
                                                  'role': 'PRIMARY'},
                                              {   'id': <DATA REMOVED>,
                                                  'role': 'SECONDARY'}],
                           'state': 'ALIVE'}} 
```

**Comment by sijeesh** [comment]
@svt-kmcdaniel , Thanks for reporting this issue.
Let me open a PR to fix this issue.

**Comment by KalpanaBhardwaj** [comment]
I have fix the issue under PR #40
